# we handle documentation distribuition in the package, so
# we don't need the builtin installation
--- libatomic-ops-1.1/Makefile.am	2005-11-25 16:19:28.000000000 +1100
+++ Makefile.am	2005-10-31 16:10:21.000000000 +1100
@@ -1,3 +1,3 @@
-SUBDIRS = src doc tests
+SUBDIRS = src tests
 
 #distclean-local:
